docs/en/reference/tools.rst

```diff
@@ -83,8 +83,6 @@ The following Commands are currently available:
    cache drivers.
 -  ``orm:clear-cache:result`` Clear result cache of the various
    cache drivers.
--  ``orm:generate-proxies`` Generates proxy classes for entity
-   classes. Deprecated in favor of using native lazy objects.
```

Comment:
A lot of the BC issues will be people still running this command somewhere (non-devs, operations people, etc.).

I'm wondering if it's worth making this a no-op, for a longer period of time?